addons/point_of_sale/models/pos_config.py

```diff
@@ -219,15 +219,12 @@ def _compute_last_session(self):
         for pos_config in self:
             session = PosSession.search_read(
                 [('config_id', '=', pos_config.id), ('state', '=', 'closed')],
-                ['cash_register_balance_end_real', 'stop_at', 'cash_register_id'],
+                ['cash_register_balance_end_real', 'stop_at', 'cash_journal_id'],
```

Comment:
Why do you need 'cash_journal_id' ?

Reply:
Nope, it is the result of replace all, it looks like `cash_register_id` was not used here